src/preface.md

```diff
@@ -24,7 +24,7 @@ learning Rust, but you might accidentally along the way.
 
 You can find all of this stuff [on GitHub](https://github.com/intermezzOS/).
 This book is in the `book` repository, the kernel is in `kernel`, and the
-website is there, too. Feel free to open issues on the [RFCs
+website is there too. Feel free to open issues on the [RFCs
```

Comment:
I'm not sure if there's an particular style you want to follow. Commas in English have weird rules that change depending on what day you happen to be talking to what person :stuck_out_tongue: